server/src/main/java/com/google/coffeehouse/storagehandler/StorageHandlerHelper.java

```diff
@@ -39,14 +40,35 @@
   * @param  clubId       the club ID string of the club we are checking the user is in
   * @return              the Boolean true or false representing if the membership exists or not
   */
-  public static Boolean checkMembership(ReadContext readContext, String userId, String clubId) {
+  public static Boolean checkAnyMembership(ReadContext readContext, String userId, String clubId) {
     Struct row =
           readContext
             .readRow(
               "Memberships",
               Key.of(userId, clubId),
               Arrays.asList("userId"));
-    return (row != null) ? true : false;
+    return (row != null);
+  }
+
+  /**
+  * Returns a Boolean that indicates whether or not a person is the owner of a club.
+  * This method creates a Struct that holds a single row from a database read row transacation.
+  * It returns false if the Struct is null, indicating the membership does not exist.
+  * It returns true if the Struct is not null, indicating the membership does exist.
```

Comment:
Consider updating the comment to reflect the additional check. We're not just checking that the Struct is non-null, we're checking that the membershipType is OWNER.

Reply:
Thank you for pointing this out, will fix that up now!